extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcConfig.java

```diff
@@ -37,6 +37,12 @@ public class OidcConfig {
     @ConfigItem
     public TokenCache tokenCache = new TokenCache();
 
+    /**
+     * Whether OIDC tenants should be resolved based on the expected Bearer token issuer `iss` claim value.
```

Comment:
```suggestion
     * If OIDC tenants should be resolved using the bearer access token's issuer (`iss`) claim value.
```
Suggesting to use `If` as as I've seen it in a few places, in `AuthConfig`, I think if we do `whether` then we should finish with `or not`, not sure... Also the Doc team likes the lower case :-) so bringing it to `bearer`